contrib/devtools/update-translations.py

```diff
@@ -97,7 +97,7 @@ def check_format_specifiers(source, translation, errors, numerus):
         errors.append("Parse error in translation for '%s': '%s'" % (sanitize_string(source), sanitize_string(translation)))
         return False
     else:
-        if source_f != translation_f:
+        if source_f is not translation_f:
```

Comment:
14903, I don't think we should change this (esp. since upstream didn't change it)